content/faq/libraries-frameworks/pdf.mdx

```diff
@@ -8,3 +8,5 @@ showToc: false
 - [ITextSharp](https://github.com/itext/itextsharp) Free only for open source project
 - [WkHtmlToPdf](https://github.com/codaxy/wkhtmltopdf) C# wrapper utility around wkhtmltopdf console tool
 - [FastReport](https://github.com/FastReports/FastReport) It can also export PDF.
+- [Stimulsoft Report](https://www.stimulsoft.com/en)(not free) It can also export PDF.
```

Comment:
Please help improving this document because telling `It can also export PDF` does not give additional value to this document. 
This is because all tools/libraries on this page have already supported PDF export.

Please also improve the benefit/description of `FastReport` as well. I know you do not add FastReport but it good to fix since we are here.

Reply:
I thinks, We should merge the Report tools and PDF generated function into same page. 
and create comparison table similar like theses. 

![image](https://user-images.githubusercontent.com/5275512/111054882-a142a580-84a2-11eb-9e24-61ebd5172659.png)

Reply:
@authapothc I like your idea. Could you please do it for us?
We can  use Markdown table. 
However, working with Markdown table can be hard because it is not easy to control multiple-lines content in each cell.

I can help you create a custom React table component and then we can it in .mdx page.

FYI, simple responsive table.
https://css-tricks.com/accessible-simple-responsive-tables/

Reply:
I think I will crate a custom React component feature comparison and then you can use it to add content.
![comparison](https://user-images.githubusercontent.com/2938310/111056544-4c0e9000-84b2-11eb-916f-756203ee8033.gif)

Reply:
Almost bro

![image](https://user-images.githubusercontent.com/2938310/111057131-8dee0500-84b7-11eb-9014-bb6c5f9326ac.png)